gen3-client/g3cmd/upload.go

```diff
@@ -103,8 +112,21 @@ func init() {
 				logs.WriteToFailedLog()
 			}
 
+			// multipart upload for large files here
+			if len(multipartFilePaths) > 0 {
+				log.Println("Multipart uploading....")
+				for _, filePath := range multipartFilePaths {
+					err = multipartUpload(uploadPath, filePath, numParallel, includeSubDirName, 0)
+					if err != nil {
+						log.Println(err.Error())
+					} else {
+						logs.IncrementScore(0)
+					}
+				}
+			}
+
 			if !logs.IsFailedLogMapEmpty() {
-				retryUpload(logs.GetFailedLogMap(), includeSubDirName, uploadPath)
+				retryUpload(logs.GetFailedLogMap(), uploadPath, numParallel, includeSubDirName)
```

Comment:
It seems that gen3-client creates a new blank record in indexd for a retry? I can not see the code to remove the old black record in indexd

Reply:
will delete any records if there is already a GUID associated with the retry object. fixed in https://github.com/uc-cdis/cdis-data-client/pull/50/commits/1cf50c4aa6f38b816b06ab8c8f0f1f74758bae3e